apps/displays/OSVR_HDK_1_3.json

```diff
@@ -0,0 +1,52 @@
+{
+  "meta": {
+    "schemaVersion": 1
+  },
+  "hmd": {
+    "device": {
+      "vendor": "OSVR",
+      "model": "HDK",
+      "num_displays": 1,
+      "Version": "1.2",
+      "Note": "Suitable for HDK 1.0-1.2"
```

Comment:
Also, the version should be 1.3 I guess.  Sorry about that.  Only changed what needed to be changed and did not look carefully at the context.